### ISSUE TITLE
Scifi dual doors not triggering anymore

### ISSUE THREAD

**Issue by S-S-X** [pandorabox-io/pandorabox.io#484]
Probably caused by this commit: https://github.com/D00Med/scifi_nodes/commit/6010bad2e964cda6bbce90f012841d53cffab669

This makes dual doors which are commonly used with fleet ships to not work well anymore as activating first door wont trigger second one anymore.

Before mentioned commit fix you could open both doors with single click.
After mentioned commit player have to activate both doors manually and of course wait for first door to open before next one cant be opened, add some lag and situation with dual doors becomes very frustrating.

No idea if there was some specific reason for this fix because mentioned commit is not linked to any issue or pull request.

Would be nice to get this dual door feature back for fleet ships.

**Comment by BobFred7** [comment]
> No idea if there was some specific reason for this fix because mentioned commit is not linked to any issue or pull request.

https://github.com/D00Med/scifi_nodes/pull/28 ... it is actually from a pull request, it seems.

**Comment by S-S-X** [comment]
Nice and thanks, could not find that myself but seems that I did not really even search.

Based on that PR the reason for change was that activating door triggered too many nodes around it.
But now situation is that not enough are triggered.

I think it should allow triggering doors directly **besides, above, below, behind and in front** of manually activated door.
Basically all 6 straight line directions from "master" door but never diagonally.

**Comment by BuckarooBanzay** [comment]
> D00Med/scifi_nodes#28 ... it is actually from a pull request, it seems.

Yes, i even waved that PR through and completely forgot that it breaks compatibility under *some* conditions...

>  Basically all 6 straight line directions from "master" door but never diagonally.

This would be perfect for the ~~double doors~~ airlocks again..
